lib/pages/home/wallet/wallet.dart

```diff
@@ -78,19 +85,9 @@ class WalletPage extends StatelessWidget {
   }
 
   Widget _buildCurrency(BuildContext context, Currency currency) {
-    if (currency.name == 'Coin' || currency.id == 1) {
```

Comment:
Don't remove the check for the Coin currency. This displays an alternative widget that properly formats Gold/Silver/Copper. For example: removing this check changes having 300 gold from displaying like: 300g, 0s, 0c to 3.000.000 gold.

Reply:
I've put this back.